.travis.yml

```diff
@@ -3,9 +3,12 @@ rvm: 2.3.1
 addons:
   codeclimate:
     repo_token: 22840649ef0d6a85c80f7276acbb377b5baee4938f1d15741f34627245c84c92
-env: RAILS_ENV=test
+env:
+  - 'RAILS_VERSION=3.2.21'
+  - 'RAILS_VERSION=4.2.6'
+  - 'RAILS_VERSION=5.0.1'
 script:
-  - bundle exec rails db:create db:migrate
+  - bundle exec rails db:drop db:create db:migrate
```

Comment:
Actually, I could try taking that out now.
The initial build was failing because the dummy app contained the same table as the engine.. but I deleted the migration that creates it.
Even though container apps will also contain that same migration, rails/rake db:migrate works for them because running migrations on a container app does not run migrations in an Engine. Travis, however, runs both and they clash. For some reason.